homework_8/participants/Competition.java

```diff
@@ -0,0 +1,13 @@
+package participants;
+
+public class Competition {
+    public static void start(Participant[] participants, Barrier[] barriers) {
+        for (Participant part : participants) {
+            for (Barrier barrier : barriers) {
+                if (!barrier.overcome(part))
+                    break;
+            }
+            System.out.println("-------------------------------------");
```

Comment:
please remove this print statement as it does not bring any meaningful value to your program, and as was mentioned, print statements should not be used in methods! You can use it in the `main` method only for testing purposes.